buildenv/jenkins/common/variables-functions

```diff
@@ -289,6 +289,13 @@ def set_job_names() {
 * Initializes all of the required variables for a Jenkins job by given job type.
 */
 def set_job_variables(job_type) {
+    // check mandatory parameters if any
+    try {
+        validate_arguments(ARGS)
+    } catch(MissingPropertyException e){
```

Comment:
Will this catch only when ARGS is not defined, or will it also catch the perror(s) we may throw inside the `validate_arguments` function?

Reply:
only when ARGS is not defined

Reply:
@AdamBrousseau - ready for review, thx